### PR TITLE
Migrate changelog to github-changelog-generator

### LINKED ISSUE #5: Migrate changelog to github-changelog-generator
This is described here: https://github.com/github-changelog-generator/github-changelog-generator#migrating-from-a-manual-changelog

This allows a fully automated changelog generation.

### PR DESCRIPTION
**Pull request by Normo**
* generate the changelog using [github-changelog-generator](https://github.com/github-changelog-generator/github-changelog-generator) and append static historical changelog `HISTORY.md`

```
github_changelog_generator -u hifis-net -p ansible-role-rsd --token <your-40-digit-github-token>  --since-tag v0.1.0
```
Closes #5

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,32 +1,24 @@
...
-### Changed
+<!--
+SPDX-FileCopyrightText: 2022 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2022 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
 
-* Implement lint and release workflow via GitHub actions
-  ([!1](https://github.com/hifis-net/ansible-role-rsd/pull/1)
-  by [Normo](https://github.com/Normo)).
+SPDX-License-Identifier: Apache-2.0
+-->
```
Comment (christianhueserhzdr) [inline CHANGELOG.md:16]: Dear Norman @Normo, now that I understood why we now have a auto-generated CHANGELOG file and a HISTORY file documenting the old state as a static historical CHANGELOG before switching to github-changelog-generator, why does it put the copyright text and license identifier here and not at the top of the file? Did we do that wrong previously or can we tell the tool to move it to the top? Does this matter at all? I have a lot of questions regarding this tool.

Comment (Normo) [reply CHANGELOG.md:16]: @christianhueserhzdr `CHANGELOG.md` is now an **auto-generated** file and I'm currently not aware of a possibility of how to tell the github-changelog-generator to add a license header to the generated changelog file.
So I decided to move the former manual changelog to `HISTORY.md` which will be automatically appended to `CHANGELOG.md` by the generator. (Migrating from a manual changelog is described [here](https://github.com/github-changelog-generator/github-changelog-generator#migrating-from-a-manual-changelog).) Of course that means the license is not at the top of the file, but `reuse` did not care where the license is located.

The alternative would be to add two files `CHANGELOG.md.license` and `HISTORY.md.license` containing the license headers.

Comment (christianhueserhzdr) [reply CHANGELOG.md:16]: Dear Norman @Normo, thanks for your explanations and ideas. I am fine with leaving the license header where it is.

**Review by christianhueserhzdr — APPROVED** [review]
Dear Norman @Normo, I approve and merge this PR now. Thanks for taking care of the CHANGELOG and HISTORY.

**Pull request merged**